doc/Type/Seq.pod6

```diff
@@ -2,15 +2,17 @@
 
 =TITLE class Seq
 
-=SUBTITLE An iterable, lazy sequence of values
+=SUBTITLE An iterable, potentially lazy sequence of values
 
     class Seq is Cool does Iterable does PositionalBindFailover { }
 
-A C<Seq> represents anything that can lazily produce a sequence of values. A
-C<Seq> is born in a state where iterating it will consume the values. However,
-calling C<.cache> on a C<Seq> will return a List that is still lazy, but stores the
-generated values for later access. However, assigning a C<Seq> to an array consumes the C<Seq>;
-alternatively, you can use the C<lazy> statement prefix to avoid it from being
+A C<Seq> represents anything that can produce a sequence of values. A
```

Comment:
Instead of eliminating lazily, why not qualify it? Use "maybe", "potentially"...

Reply:
I believe an introductory sentence should be short, describing the core properties. Laziness is a bit of a problematic concept:

* in the sense of `.is-lazy` returning `True`, it is an option that is not the default
* in the sense of [lazy evaluation](https://en.wikipedia.org/wiki/Lazy_evaluation), only a `Seq` that is cached can be considered lazy because values are not _shared_ if the sequence is not cached

Maybe it's a good idea to point this out after introducing `.cache` and to consistently use "`.is-lazy`" instead of "is lazy" in all cases where laziness refers only to the first sense.  What do you think?

Reply:
OK, fair enough.